cryptographic_fields/fields.py

```diff
@@ -25,30 +29,37 @@
 crypter = cryptography.fernet.MultiFernet(keys)
 
 
-def encrypt_str(s):
-    # be sure to encode the string to bytes
+def encrypt_str(s, salt):
+    s = u'{salt}:{string}'.format(
```

Comment:
@LucasRoesler This is actually dangerous, since the ```:``` character might show up in the salted bytes. We should make sure to replace that character if it appears before generated the ```salt + string``` value that is encrypted.